test/v1/services/DeletePensionChargesServiceSpec.scala

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.services
+
+import uk.gov.hmrc.domain.Nino
+import v1.mocks.connectors.MockPensionChargesConnector
+import v1.models.errors._
+import v1.models.outcomes.DesResponse
+import v1.models.requestData.{DeletePensionChargesRequest, DesTaxYear}
+
+import scala.concurrent.Future
+
+class DeletePensionChargesServiceSpec extends ServiceSpec {
+
+  val correlationId = "a1e8057e-fbbc-47a8-a8b4-78d9f015c253"
+
+  val nino = Nino("AA123456A")
+  val taxYear = DesTaxYear("2019-20")
```

Comment:
While this will work as it is not hitting validation, could we make this 2020-21 as that is the minimum tax year and would help with consistency of data through app?